cli/src/opts/cast.rs

```diff
@@ -424,37 +424,48 @@ Defaults to decoding output data. To decode input data pass --input or use cast
     },
     #[clap(name = "4byte")]
     #[clap(aliases = &["4", "4b"])]
-    #[clap(about = "Get the function signatures for the given selector from 4byte.directory.")]
+    #[clap(
+        about = "Get the function signatures for the given selector from https://sig.eth.samczsun.com."
+    )]
     FourByte {
         #[clap(help = "The function selector.")]
         selector: String,
     },
     #[clap(name = "4byte-decode")]
     #[clap(aliases = &["4d", "4bd"])]
-    #[clap(about = "Decode ABI-encoded calldata using 4byte.directory.")]
+    #[clap(about = "Decode ABI-encoded calldata using https://sig.eth.samczsun.com.")]
     FourByteDecode {
         #[clap(help = "The ABI-encoded calldata.")]
         calldata: String,
     },
     #[clap(name = "4byte-event")]
     #[clap(aliases = &["4e", "4be"])]
-    #[clap(about = "Get the event signature for a given topic 0 from 4byte.directory.")]
+    #[clap(
+        about = "Get the event signature for a given topic 0 from https://sig.eth.samczsun.com."
+    )]
     FourByteEvent {
         #[clap(help = "Topic 0", value_name = "TOPIC_0")]
         topic: String,
     },
+    #[clap(name = "upload-signature")]
+    #[clap(aliases = &["ups"])]
+    #[clap(about = "Upload the given signatures to https://sig.eth.samczsun.com.")]
```

Comment:
should we add an example in the about section?

Reply:
done in https://github.com/foundry-rs/foundry/pull/1716/commits/5e4c84b1603f39bf69ebba0e076f94e80ec87f46